sample/build.gradle

```diff
@@ -9,7 +9,8 @@ dependencies {
   implementation libs.squareup.okhttp.logging.interceptor
 
   testImplementation libs.junit
-  testImplementation 'app.cash.molecule:molecule-testing'
+  testImplementation 'org.jetbrains.kotlinx:kotlinx-coroutines-test:1.6.4'
+  testImplementation 'app.cash.turbine:turbine:0.9.0'
```

Comment:
Please use the version catalog for both of these dependencies.

Reply:
Sure, I just added the change as well